examples/rondpoint/tests/bindings/test_rondpoint.kts

```diff
@@ -6,7 +6,15 @@ assert(dico == copyDico)
 
 assert(copieEnumeration(Enumeration.DEUX) == Enumeration.DEUX)
 assert(copieEnumerations(listOf(Enumeration.UN, Enumeration.DEUX)) == listOf(Enumeration.UN, Enumeration.DEUX))
-assert(copieCarte(mapOf("1" to Enumeration.UN, "2" to Enumeration.DEUX)) == mapOf("1" to Enumeration.UN, "2" to Enumeration.DEUX))
+assert(copieCarte(mapOf(
+    "0" to EnumerationAvecDonnees.ZERO(),
+    "1" to EnumerationAvecDonnees.UN(1),
+    "2" to EnumerationAvecDonnees.DEUX(2, "deux")
+)) == mapOf(
+    "0" to EnumerationAvecDonnees.ZERO(),
+    "1" to EnumerationAvecDonnees.UN(1),
+    "2" to EnumerationAvecDonnees.DEUX(2, "deux")
+))
```

Comment:
(I haven't actually run any of these examples, just trying my best to sketch out what the resulting code would actually look like to use).

Reply:
In Kotlin, since they're inner classes, they'd be spelled with CamelCase. 

They'd be defined as:

```kotlin
sealed class EnumerationAvecDonnees {
    object Zero: EnumerationAvecDonnees() // object for noargs
    data class Un(val value: Int): EnumerationAvecDonnees()
    data class Deux(val value: Int, val nombre: String): EnumerationAvecDonnees()
}

assert(EnumerationAvecDonnees.Zero != EnumerationAvecDonnees.Deux(2, "two"))
```

Reply:
> (in retrospect, this comment feels like "Well Actually". Sorry)

Heh, no worries at all, exactly the sort of early feedback I'm after 👍